bootstrap.yml

```diff
@@ -1836,3 +1836,24 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
+
+  - name: lua
+    source:
+      subdir: ports
+      url: 'https://www.lua.org/ftp/lua-5.3.5.tar.gz'
```

Comment:
Can we pull from a git repo, or is there no official one?

Reply:
Well, we could use https://github.com/lua/lua, but for me the build config looks bad because Makefile in git is different from tar.gz.

The build config from git looks like this:
```yml
- args:
  - 'make'
  - 'CC=x86_64-managarm-gcc'
  - 'AR=x86_64-managarm-ar rcu'
  - 'RANLIB=x86_64-managarm-ranlib'
  - 'MYCFLAGS='
  - 'MYLIBS='
  - '-j@PARALLELISM@'
- args: ['mkdir', '-p', '@THIS_COLLECT_DIR@/bin']
- args: ['install', '-m', '0755', 'lua', '@THIS_COLLECT_DIR@/bin']
```